homeassistant/components/n26/sensor.py

```diff
@@ -0,0 +1,288 @@
+        self._account_statuses = self._client.get_account_statuses()
+        self._limits = self._client.get_account_limits()
+        self._balance = self._client.get_balance()
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the entity."""
+        return self._balance["iban"]
+
+    @property
+    def name(self) -> str:
+        """Friendly name of the sensor."""
+        return "n26_%s" % self._balance["iban"].lower()
+
+    @property
+    def state(self) -> float:
+        """Return the balance of the account as state."""
+        return self._balance["bankBalance"]
+
+    @property
+    def unit_of_measurement(self) -> str:
+        """Use the currency as unit of measurement."""
+        return self._balance["currency"]
+
+    @property
+    def device_state_attributes(self) -> dict:
+        """Additional attributes of the sensor."""
+        attributes = {
+            ATTR_IBAN: self._balance.get("iban"),
+            ATTR_AVAILABLE_BALANCE: self._balance.get("availableBalance"),
+            ATTR_USABLE_BALANCE: self._balance.get("usableBalance"),
+            ATTR_ACC_OWNER_TITLE: self._account_info.get("title"),
+            ATTR_ACC_OWNER_FIRST_NAME: self._account_info.get("kycFirstName"),
+            ATTR_ACC_OWNER_LAST_NAME: self._account_info.get("kycLastName"),
+            ATTR_ACC_OWNER_GENDER: self._account_info.get("gender"),
+            ATTR_ACC_OWNER_BIRTH_DATE: timestamp_ms_to_date(
+                self._account_info.get("birthDate")),
+            ATTR_ACC_OWNER_EMAIL: self._account_info.get("email"),
+            ATTR_ACC_OWNER_PHONE_NUMBER:
+                self._account_info.get("mobilePhoneNumber"),
+        }
+
+        for limit in self._limits:
+            limit_attr_name = ("limit_%s" % limit["limit"]).lower()
+            attributes[limit_attr_name] = limit["amount"]
+
+        return attributes
+
+    @property
+    def icon(self) -> str:
+        """Set the icon for the sensor."""
+        return ICON_ACCOUNT
+
+
+class N26Card(Entity):
+    """Sensor for a N26 card."""
+
+    def __init__(self, client, _account_name, card) -> None:
+        """Initialize a N26 card."""
+        self._client = client
+        self._account_name = _account_name
+        self._card = card
+
+    @property
+    def should_poll(self) -> bool:
+        """Return True.
+
+        Data needs to be polled from the bank servers.
+        """
+        return True
+
+    def update(self) -> None:
+        """Get the current balance and currency for the account."""
+        for card in self._client.get_cards():
+            if card["id"] == self._card["id"]:
+                self._card = card
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the entity."""
+        return self._card["id"]
+
+    @property
+    def name(self) -> str:
+        """Friendly name of the sensor."""
+        return "%s_card_%s" % (self._account_name.lower(),
```

Comment:
This is the text displayed in the UI, you don't need replace "-"

Reply:
done